### ISSUE TITLE
Python package && command

### ISSUE THREAD

**Issue by marcosschroh** [marcosschroh/pyinstamation#30]
**Is your feature request related to a problem? Please describe.**
Have a python package and a script to run the bot.

**Describe the solution you'd like**
Right now we hace to clone the project, create the `config.yml` and `make run-bot`

Maybe we can create a python package, and create a command to run the bot?

Summary should be:

`pip install pyinstamation && pyinstamation run --config=path-to-my-config.yml`

**Comment by woile** [comment]
The problem is the dependency with selenium. I think the best would be to have a docker published.

**Comment by marcosschroh** [comment]
Do you mean the dependency about specifing the driver location (`DRIVER_LOCATION_CHROME` and `DRIVER_LOCATION_FIREFOX`)? If that is the case, could be a config option as well? 

The docker option is good as well.

**Comment by woile** [comment]
Yes, I think the doker will be the best way to distribute the application, it will just work.

We can make pip installable a script requiring one of this 2 options:
1. specify the path of the driver in an ENV. Example: `SELENIUM_DRIVER="~/.pyinstamation/chrome-driver"`
2. Passing the driver as a paramater. E.g: `pyinstamation --driver ~/.pyinstamation/chrome-driver`.
